CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.1 November 26, 2021
+
+## Bug fixes 🐞
+* Fix billing issue when upgrading Mapbox Maps SDK from v9 to v10.
```

Comment:
was the issue only when you upgrade from hydrogen to carbon, or was it for older releases (eg v6 or something else) to carbon, just curious?

Reply:
nit: add link to the PR as well?

Reply:
I am not adding links here as there's one PR for it all 😄 
I'll add it though to release notes.